phoenicis-library/src/main/java/org/phoenicis/library/ShortcutRunner.java

```diff
@@ -43,24 +43,30 @@ public void run(ShortcutDTO shortcutDTO, List<String> arguments, Consumer<Except
         final InteractiveScriptSession interactiveScriptSession = scriptInterpreter.createInteractiveSession();
 
         interactiveScriptSession.eval("include(\"engines.wine.shortcuts.reader\");",
-                ignored -> interactiveScriptSession.eval("new ShortcutReader()", output -> {
-                    final Value shortcutReader = (Value) output;
+                result -> {
```

Comment:
This code is the same everywhere. Can we extract it to a separate method?

Reply:
My idea was to do this later on in a centralized method in `PhoenicisScriptEngine`:

```java
public interface PhoenicisScriptEngine {
    <E> E evalAndReturn(String script, Class<E> type);
    
    <E> void eval(String script, Class<E> type, Consumer<E> onSuccess, Consumer<Exception> onError);
}
```

See also our [documentation](https://phoenicisorg.github.io/phoenicis/Developers/script-interaction/).

For now I would suggest to retain the redundancy and fix this later for all script types (i.e. engines, installers, plugins, shortcuts, verbs etc.) at once.

Reply:
I'm fine with that.